resources/views/oneclick/mall_diferido/start_inscription.blade.php

```diff
@@ -0,0 +1,18 @@
+<h1>Oneclick inscripción</h1>
+
+
+
+<form method="post" action="startInscription"  style="display: flex; flex-direction:column; width:50%;font-size: 20px;">
+    @csrf
+    <label for="userName">Nombre de usuario</label>
+    <input id="userName" name="user_name" value="nombre_de_usuario">
+
+    <label for="email">Email</label>
+    <input type="email" id="email" name="email" value="example@example.com">
+
+    <label for="responseUrl">URL de Respuesta</label>
+    <input id="responseUrl" name="response_url" value="http://localhost:8000/oneclick/responseUrl">
+
+
+    <button type="submit">Enviar datos</button>
```

Comment:
the PR name said Patpass status but this is for oneclick. can you remove it or update the PR title and description.